master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -1,45 +1,77 @@
 ---
-title: Kubeadm Hosted Install
+title: kubeadm Hosted Install
 ---
 
-This document outlines how to install Calico, as well as a as single node
-etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm.
+This document outlines how to install Calico on a kubeadm cluster.
 If you have already built your cluster with kubeadm, please review the
 [Requirements / Limitations](#requirements--limitations) at the bottom of
 this page. It is likely you will need to recreate your cluster with the
 `--pod-network-cidr` and `--service-cidr` arguments to kubeadm.
 
-Users who have deployed their own etcd cluster outside of kubeadm should
-use the [Calico only manifest](../hosted) instead, as it does not deploy its
-own etcd.
+## Installation
+
+You can easily create a cluster compatible with these manifests by following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
+
+Calico installations require kubeadm initialized clusters to have set a `--pod-network-cidr`
+which should match the value set by `CALICO_IPV4POOL_CIDR` in the manifest:
+
+```yaml
+- name: CALICO_IPV4POOL_CIDR
+  value: "192.168.0.0/16"
+```
+
+This value can be changed to any CIDR that does not overlap the clusters service CIDR.
+The manifests are currently setup to use the CIDR `192.168.0.0/16` so you would initialize
+the cluster with the following command:
 
-You can easily create a cluster compatible with this manifest by following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
+```shell
+kubeadm init --pod-network-cidr=192.168.0.0/16
+```
 
+### etcd datastore
 
-#### Installation
+Users who have deployed their own etcd cluster outside of kubeadm should
+use the [Calico only manifest](../hosted) instead, as it does not deploy its
+own etcd.
 
 To install this Calico and a single node etcd on a run the following command
 depending on your kubeadm / kubernetes version:
 
-For Kubeadm stable with Kubernetes version >= v1.6.0:
+For kubeadm stable with Kubernetes version >= v1.6.0:
 
-```
+```shell
 kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubeadm/1.6/calico.yaml
 ```
 
 >[Click here to view the above yaml directly.](1.6/calico.yaml)
 
-For Kubeadm 1.5 with Kubernetes version v1.5.x:
+For kubeadm 1.5 with Kubernetes version v1.5.x:
```

Comment:
strictly speaking we can remove this bit - we don't need to support v1.5 in our docs any more.

Reply:
happy for that be a new PR though.